invoke.example.yml

```diff
@@ -1,10 +1,10 @@
 ---
 nautobot_dev_example:
   project_name: "nautobot-dev-example"
-  nautobot_ver: "2.0.0"
+  nautobot_ver: "2.3.1"
   local: false
   python_ver: "3.11"
-  compose_dir: "development"
+  compose_dir: "/full/path/to/nautobot-app-dev-example/development"
```

Comment:
Why is the full path necessary?

Reply:
I am able to use relative paths in another repo where I have a custom invoke.yml.

Reply:
I personally don't ever use invoke commands from anything other than the root. Having to change that is worse for my workflow. 

Not losing sleep on it, but just my preference is to keep it relative.

Reply:
Out of curiosity, what's the use case for overriding `compose_dir` in `invoke.yml` in any case?

Reply:
That line is commented out now. I don't know of any use case for overriding it.